kotlin/android/app/src/main/java/dev/firezone/android/features/session/ui/SessionViewModel.kt

```diff
@@ -15,6 +15,7 @@ internal class SessionViewModel
     constructor() : ViewModel() {
         @Inject
         internal lateinit var repo: Repository
+        val favoriteResources = MutableLiveData<HashSet<String>>()
```

Comment:
I think this should be private and we expose a getter and setter instead? AFAIK that is the Java / Kt convention. We do the same for `resourcesLiveData`.